src/index.js

```diff
@@ -1,33 +1,24 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./sass/main.scss";
-import Month from "./component/Month";
+import DatePicker from "./component/DatePicker";
 import Header from "./component/Header";
 import Footer from "./component/Footer";
 
 const App = () => {
-	const [focused, setFocused] = React.useState(false);
+	
+	const [isFocused, setFocused] = React.useState(false);
 
-	const handleFocusOut = (e) => {
-		e.preventDefault();
-		e.stopPropagation();
-
-		const className = e.target.classList[0];
-
-		if (
-			className &&
-			className.slice(0, 4) !== "grid" &&
-			className.slice(0, 10) !== "datePicker"
-		) {
-			setFocused(false);
-		}
-	};
 	return (
-		<div onClick={handleFocusOut} className="App">
+		<div className="App">
 			<Header />
-			<Month focused={focused} setFocused={setFocused} />
+			<DatePicker 
+				onFocusChange={({focused}) => setFocused(focused)} 
```

Comment:
This is a generally accepted prop to have on a datePicker - see [this airbnb picker](https://github.com/airbnb/react-dates#singledatepicker)